src/raygui.h

```diff
@@ -3355,122 +3355,32 @@ int GuiListViewEx(Rectangle bounds, const char **text, int count, int *scrollInd
     return result;
 }
 
-// Color Panel control
+// Color Panel control - Color (RGBA) variant.
 int GuiColorPanel(Rectangle bounds, const char *text, Color *color)
 {
     int result = 0;
-    GuiState state = guiState;
-    Vector2 pickerSelector = { 0 };
-
-    const Color colWhite = { 255, 255, 255, 255 };
-    const Color colBlack = { 0, 0, 0, 255 };
 
     Vector3 vcolor = { (float)color->r/255.0f, (float)color->g/255.0f, (float)color->b/255.0f };
     Vector3 hsv = ConvertRGBtoHSV(vcolor);
+    Vector3 prev_hsv = hsv; // workaround to see if GuiColorPanelHSV modifies the hsv.
```

Comment:
Please, could you review naming to follow `camelCase` convention?

Reply:
Yes of course! My apologies, it is sometimes difficult to keep all conventions in mind when switching between projects.